### DISCUSSION TITLE
Add other color variants for radial progress

### DISCUSSION THREAD

**Discussion by henacodes** [saadeghi/daisyui#2362] [Ideas / feature requests]
There is only 'text-primary' color variant for radial progress. add other color variants for it

**Comment by saadeghi** [comment]
All colors work (both Tailwind CSS color names + daisyUI color names) So it can be anything (`text-secondary`, `text-success`, `text-green-600`,...)
The example on the site just shows how you can use any custom color using Tailwind CSS `bg-*` or `text-*` color class names.

**Reply by henacodes** [reply]
Ohhh sir 
I am really sorry.  Now, it worked after restarting the server
